go/chat/server.go

```diff
@@ -1045,6 +1045,34 @@ func (h *Server) PostMetadata(ctx context.Context, arg chat1.PostMetadataArg) (c
 	return h.PostLocal(ctx, parg)
 }
 
+func (h *Server) PostDeleteHistory(ctx context.Context, arg chat1.PostDeleteHistoryArg) (res chat1.PostLocalRes, err error) {
```

Comment:
We might want to make two of these, one that takes age, and another that takes a message ID. I think there is talk of just adding an option to delete on the messages themselves.